sigtrap_posix.go

```diff
@@ -90,6 +90,7 @@ func trapSignalsPosix() {
 				purgeEventHooks()
 
 				// Kick off the restart; our work is done
+				EmitEvent(InstanceRestart, nil)
```

Comment:
You missed this one 😛 needs to be `InstanceRestartEvent` too

Reply:
oops, fixed :)